assets/src/edit-story/components/carousel/secondaryMenu.js

```diff
@@ -116,7 +121,9 @@ function SecondaryMenu() {
         <Space />
         {enableChecklistCompanion && (
           <>
-            <Checklist />
+            <ChecklistCountProvider>
+              <Checklist />
+            </ChecklistCountProvider>
```

Comment:
needed to move this provider higher so we could calculate the number of visible panels in the checklist. This is used to calculate the max height of the panel so that we can scroll it.

Reply:
this might mean we should follow up with a ticket to combine the count and the checklistContext since that's just barely above this.

Reply:
I like that idea! One big checklist context makes sense.